### PR TITLE
fixes related to #1217

### PR DESCRIPTION
**Pull request by EliAndrewC**
This is the MagClassic code that changed resulting from the fixes made at
https://github.com/magfest/ubersystem/pull/1224

### CODE REVIEW

File: magclassic/templates/regextra.html [hunk 1]
```diff
@@ -1,12 +1,16 @@
 
 <script>
     if (window.BADGE_TYPES) {
-        window.BADGE_TYPES[0].description = 'Allows access to the festival for its duration.';
+        window.BADGE_TYPES.options[0].description = 'Allows access to the festival for its duration (Sept 11-13).';
     }
 
     $(function() {
         $('#tshirt-size-guide').insertAfter($.field('shirt'));
```
Comment (EliAndrewC) [inline magclassic/templates/regextra.html]: This is an unrelated fix; because many of our forms don't have group fields, we only do the move if it exists and otherwise remove the explanation.

**Comment by kitsuta** [comment]
+1
